pkg/proxy/userspace/roundrobin.go

```diff
@@ -356,6 +358,11 @@ func (lb *LoadBalancerRR) OnEndpointsDelete(endpoints *v1.Endpoints) {
 }
 
 func (lb *LoadBalancerRR) OnEndpointsSynced() {
+	close(lb.endpointsSynced)
```

Comment:
Why do we have 2 different mechanisms for being notified of endpoint sync?

Reply:
@thockin userspace proxy doesn't implement EndpointsHandler, the LB does. So we have to somehow notify the userspace proxy of when the endpoints have been synced.

However, I guess we could just have the userspace proxy implement EndpointsHandler and pass the calls through to the load balancer object that it was originall given? That's a larger code change though.

Reply:
Implemented EndpointsHandler passthrough.